js/viz.js

```diff
@@ -240,6 +311,10 @@ const viz = {
     return node;
   },
 
+  dragStart() {
+    d3.event.subject.shadow = true;
```

Comment:
The current node highlight behaviour is as follows:
- Drag a node
- This is the active node and corresponding node has a shadow
- At drag end, this node isn't active anymore, and the shadow goes away

This behaviour is easy, as I know the active node via `d3.event.subject`.

@jonathanKingston @biancadanforth Do we need to highlight the node even after the drag end? (This is the behaviour in the old version)

In this case, we will have to do the following:
```
for(node in this.nodes) {
 node.shadow = false;
 if (node.hostname === d3.event.subject.hostname) {
   node.shadow = true;
 }
}
```
I don't want to iterate over all the nodes, this can be expensive? @jonathanKingston inputs welcome